update-repo.sh

```diff
@@ -132,7 +132,11 @@ else
     )
     diff_uri="${base_uri}/compare/${old_tag}...${new_tag}"
```

Comment:
[optional ]Maybe this could done inside the else block as it seems `diff_uri` is only used there.